src/test/scala/cornichonTests/FeatureWithProduct.scala

```diff
@@ -0,0 +1,119 @@
+package cornichonTests
+
+import com.github.agourlay.cornichon.CornichonFeature
+
+trait FeatureWithProduct {
+  this: CornichonFeature =>
+
+  def AddProduct =
```

Comment:
```suggestion
  def addProduct =
```
to be consistent with other methods?

Reply:
indent of json fixed in e2e309d